### PR TITLE
fix: [STO-4611]: added security tests tab to module

### PR DESCRIPTION
**Pull request by atmollo-harness**
##### Screenshots:

image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionTabs/ExecutionTabs.tsx [hunk 1]
```diff
@@ -221,7 +222,7 @@ export default function ExecutionTabs(props: ExecutionTabsProps): React.ReactEle
     })
   }
 
-  if ((isCD && stoCDPipelineSecurityEnabled) || (isCI && stoCIPipelineSecurityEnabled)) {
+  if ((isCD && stoCDPipelineSecurityEnabled) || (isCI && stoCIPipelineSecurityEnabled) || isSTO) {
```
Comment (chris-harness) [inline src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionTabs/ExecutionTabs.tsx:225]:
```suggestion
  if ((isCD && stoCDPipelineSecurityEnabled) || ((isCI || isSTO) && stoCIPipelineSecurityEnabled)) {
```

**Comment by atmollo-harness** [comment]
retrigger titlecheck

**Comment by atmollo-harness** [comment]
retrigger titlecheck

**Comment by atmollo-harness** [comment]
retrigger titlecheck

**Review by r0610205 — APPROVED** [review]

**Comment by atmollo-harness** [comment]
retrigger titlecheck

**Review by atmollo-harness — COMMENTED** [review]
fix: [STO-4611]: sync PRs

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionTabs/ExecutionTabs.tsx [hunk 2]
```diff
@@ -221,7 +222,7 @@ export default function ExecutionTabs(props: ExecutionTabsProps): React.ReactEle
     })
   }
 
-  if ((isCD && stoCDPipelineSecurityEnabled) || (isCI && stoCIPipelineSecurityEnabled)) {
+  if ((isCD && stoCDPipelineSecurityEnabled) || ((isCI || isSTO) && stoCIPipelineSecurityEnabled)) {
```
[the remark this answers is not in the document]
Comment (atmollo-harness) [reply src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionTabs/ExecutionTabs.tsx:225]: This one is right the hotfix PR should now match this change

**Review by chris-harness — APPROVED** [review]

**Pull request merged**